db/src/tx.rs

```diff
@@ -677,6 +689,12 @@ impl<'conn, 'a, W> Tx<'conn, 'a, W> where W: TransactWatcher {
                         (true, true) => fts_many.push(reduced),
                     }
                 },
+
+                Term::RetractEntity(KnownEntid(e)) => {
+                    // TODO: Might update metadata?
```

Comment:
We shouldn't allow retracting vocabulary, so this might update idents… but that's a can of worms.